pylot.py

```diff
@@ -242,8 +246,12 @@ def add_planning_component(graph,
                            goal_location,
                            carla_op,
                            agent_op):
-    planning_op = pylot.operator_creator.create_planning_op(
-        graph, goal_location)
+    if FLAGS.waypoint_planning_operator:
+        planning_op = pylot.operator_creator.create_waypoint_planning_op(graph, goal_location)
```

Comment:
Does the MPCAgentOperator work with the normal PlanningOperator that we have? Do we require the WaypointPlanningOperator?

Reply:
See: https://github.com/erdos-project/pylot/pull/37#discussion_r349956067